src/locales/en.ts

```diff
@@ -2,44 +2,44 @@ import { RomcalLocale } from '@romcal/models/locale/romcal-locale.type';
 
 export default {
   advent: {
-    season: 'Advent',
+    season: 'Advent Season',
     weekday: '{{day}} of the {{week}} week of Advent',
     sunday: '{{week}} Sunday of Advent',
   },
   christmastide: {
-    season: 'Christmas',
+    season: 'Christmas Season',
     day: '{{day}} of Christmastide',
-    octave: '{{count}} day in the Octave of Christmas',
+    octave: '{{count}} Day within the Octave of the Nativity of the Lord',
     sunday: '{{count}} Sunday of Christmas',
   },
   epiphany: {
```

Comment:
@emagnier, is the `epiphany` ‘season’ still part of romcal? We have talked about merging it into the `christmastide` season (which it is part of).

Reply:
This isn't a season, but still a period (within the season of Christmas).

Reply:
Okay.